app/src/main/res/layout/activity_websearch.xml

```diff
@@ -0,0 +1,105 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This file is unnecessary. It was refactored in my last commit and you have probably included this because your fork was not properly synced.
Please remove it.